csrc/ir/nodes.cpp

```diff
@@ -5756,4 +5765,177 @@ std::vector<PolymorphicValue> TopKOp::evaluate(
 
 NVFUSER_DEFINE_CLONE_AND_CREATE(TopKOp)
 
+GroupedMmaOp::GroupedMmaOp(
+    IrBuilderPasskey passkey,
+    Val* out,
+    Val* mat1,
+    Val* mat2,
+    Val* offsets,
+    Val* scale1,
+    Val* scale2)
+    : Expr(passkey) {
+  NVF_ERROR(
+      out->getValType().value() == ValType::TensorView,
+      "Output must be a TensorView");
+  NVF_ERROR(
+      mat1->getValType().value() == ValType::TensorView,
+      "First input must be a TensorView");
+  NVF_ERROR(
+      mat2->getValType().value() == ValType::TensorView,
+      "Second input must be a TensorView");
+  NVF_ERROR(
+      offsets->getValType().value() == ValType::TensorView,
+      "Offsets must be a TensorView");
+  addOutput(out);
+  addInput(mat1);
+  addInput(mat2);
+  addInput(offsets);
+
+  bool has_scale1 = scale1 != nullptr;
+  if (has_scale1) {
+    NVF_CHECK(
+        scale1->getValType().value() == ValType::TensorView,
+        "Scale1 must be a TensorView");
+    NVF_CHECK(
+        scale2->getValType().value() == ValType::TensorView,
+        "Scale2 must be a TensorView");
+    addInput(scale1);
+    addInput(scale2);
+  }
+  addDataAttribute(has_scale1);
+}
+
+std::string GroupedMmaOp::toString(int indent_size) const {
+  std::stringstream ss;
+  indent(ss, indent_size) << out()->toString() << " = GroupedMmaOp("
```

Comment:
```suggestion
  indent(ss, indent_size) << out() << " = GroupedMmaOp("
```
as long as `#include <ir/iostream.h>`